evaluator/notifications.go

```diff
@@ -81,7 +81,16 @@ func publishNewAdvisoriesNotification(tx *gorm.DB, system *models.SystemPlatform
 		events = append(events, ntf.Event{Payload: advisory, Metadata: ntf.Metadata{}})
 	}
 
-	notif, err := ntf.MakeNotification(system, event, NewAdvisoryEvent, events)
+	var orgID string
+	err = tx.Table("rh_account acc").Select("acc.org_id").
+		Joins("JOIN system_platform sp on sp.rh_account_id=acc.id").
+		Where("sp.inventory_id", system.InventoryID).
+		Take(&orgID).Error
+	if err != nil {
+		return errors.Wrap(err, "querying orgID failed")
+	}
+
+	notif, err := ntf.MakeNotification(system, event, orgID, NewAdvisoryEvent, events)
```

Comment:
why query it always when most of the time you have orgID? Query it in vmaas-sync and send it in re-evaluation message

Reply:
It will be even more spaghetti code, but no problem for me.